### ISSUE TITLE
-t option for irsync

### ISSUE THREAD

**Issue by phuongdm** [irods/irods#4360]
## Feature

Hi, is there any feature like `-t` of rsync on irsync? It will safe us hours/days for backup data.

Thanks!

**Comment by JustinKyleJames** [comment]
I do not believe there is.  You might  use the --age to help but it is not the same.


________________________________

**Comment by phuongdm** [comment]
Is there any plan to add this option into irsync on next release? and also with --delete. You may feel the power of these options while working with 100,000+ files/dirs and slow connection :(

**Comment by trel** [comment]
partial duplicate of #2780
